src/main/scala/nl/knaw/dans/easy/multideposit/package.scala

```diff
@@ -255,6 +255,20 @@ package object multideposit {
         value2 <- value.toOption
       } yield value2
     }
+
```

Comment:
Some more documentation on these 4 methods wouldn't hurt IMHO. As both `Dataset` and `Dictionary` are somewhat more complex data structures, this documentation would get the functionality of these methods across a little faster to the others.